components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -123,9 +123,6 @@ void KeyringController::Lock() {
 }
 
 bool KeyringController::Unlock(const std::string& password) {
-  if (!default_keyring_)
```

Comment:
I think you have to call `ResumeDefaultKeyring` for every browser restart instead of removing this check.

Reply:
per discussion on slack, let's make `ResumeDefaultKeyring` private and update the unit test

Reply:
Or just move it into `Unlock`

Reply:
Addressed in 914edcd0743bb11fbd828e7b946ba4e357a5caac